bin/foreman-proxy-certs-generate

```diff
@@ -16,10 +16,10 @@ Kafo::KafoConfigure.hooking.register_pre(:init) do
     if data && data[:answer_file] && File.file?(data[:answer_file])
       scenario = YAML.load_file(data[:answer_file])
       organization = begin
-                       scenario['foreman']['initial_organization']
-                     rescue StandardError
-                       nil
-                     end
+        scenario['foreman']['initial_organization']
+      rescue StandardError
+        nil
+      end
```

Comment:
You can use [Hash#dig](https://www.rubydoc.info/stdlib/core/Hash:dig) together with the same navigator operator to safely get the value:
```suggestion
      organization = scenario&.dig('foreman', 'initial_organization')
```